src/components/Forms/UserRole/UserRole.tsx

```diff
@@ -29,12 +35,29 @@ function UserRoleForm(props: UserRoleProps) {
 	let initialValues: IUserRole = getInitialValues(props);
 	const dashboardData = useDashBoardData();
 	const formAction = props.type;
-	//
-	const { data: role } = useQuery(GET_ROLES_BY_ORG, {
+	const [
+		sendInvitationToUser,
+		{ data: sendInvitationToUserResponse, loading: sendInvitationToUserLoading },
+	] = useMutation(INVITE_USER, {
+		onCompleted(data) {
+			notificationDispatch(setSuccessNotification("Invitation Sent"));
+		},
+		onError(err) {
+			notificationDispatch(setErrorNotification("Inviting User Failed !"));
+		},
+	});
+	useQuery(GET_ROLES_BY_ORG, {
 		variables: { filter: { organization: dashboardData?.organization?.id } },
 		onCompleted(data) {
-			if (data?.roles) {
-				userRoleForm[1].optionsArray = data.roles;
+			if (data?.organizationRoles) {
+				let roleArr: any = [];
+				data.organizationRoles.forEach((role: any) => {
+					/*excluding Admin role here from roles*/
+					if (role.type !== `admin-org-${dashboardData?.organization?.id}`) {
```

Comment:
why are we match role type with a string directly?

Reply:
this is to remove admin role from list .. every role contains this type i.e admin-org-[organizationID] . i discussed this with amresh sir